content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+
+For e.g., the anatomy of a component card and it's different types can be found on the [UI Component > Card page](https://o3de.org/docs/tools-ui/component-library/uidev-card-component/), while the what contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found on the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
+
+![Preview image for the O3DE component card guidelines](/images/blog/intro-to-bjds/componentcard-ux-preview.jpg)
+
+Currently we have two UX Pattern pages on O3DE Learn: Component Card Properties and Writing UI Message guidelines. We plan to roll out more pages in the future coming months, so stay tuned.
+
+#### O3DE Branding Guidelines
+
+Our [brand guidelines section](https://o3de.org/docs/tools-ui/branding-guidelines/) includes information on how to apply the O3DE brand, and downloadable assets such as; a PDF style guide, logos files and fonts etc. These pages are especially important if you are looking to represent the O3DE logo in your product or marketing website. 
+
+![Preview image for the O3DE branding guidelines](/images/blog/intro-to-bjds/branding-guidelines-preview.jpg)
+
+If there's an asset that's missing that you're looking for, don't hesitate to [file a ticket](https://github.com/o3de/sig-ui-ux/issues/new/choose) and the community will help get the right asset you need. We are always happy to help!
+### BlueJay Design System's Roadmap
+
+![Diagram showing the tenets of the O3DE design system](/images/blog/intro-to-bjds/roadmap2.jpg)
+
+Above is our planned roadmap for 2021–2022. These timeframes are estimates based off our current SIG-UI-UX contributions rate. Times may vary; if we are able to gather more community support and help, items could be completed even sooner.
+
+For a full detailed breakdown on our roadmap, you can read our article here.
+
+#### Looking Forward
+
+The next steps for BJDS involve planning out resources and commitments towards achieving these goals. As Phase 2 comes to a close soon, Phase 3 is will be achievable with the help of the SIG-UI-UX community effort as a whole. As a core contributor towards this design system, I will be looking to help kick start some of these new pages to get the ball going. 
+
+If you would like to help contribute towards the efforts here—by reviewing, writing or letting us know to remind you when we want some feedback—let us know on [Discord](https://discord.com/invite/o3de)! All the extra eyes and hands are a massive help.
+
+You can see here that the BJDS has big dreams and ambitions. As a core, it's just a design system for an open source 3D Engine. But as an end goal, we want this to be much more than that. 
```

Comment:
If you're speaking generally about an "open-source, 3D engine", then use the adjective form ("open-source") with a hypen, and add a comma between the two adjectives. If you're talking about "Open 3D Engine", capitalize the whole thing. 

```suggestion
You can see here that the BJDS has big dreams and ambitions. At its core, it's just a design system for an open-source, 3D engine. But as an end goal, we want this to be much more than that. 
```